backend/apps/webui/models/users.py

```diff
@@ -114,9 +120,23 @@ def get_user_by_api_key(self, api_key: str) -> Optional[UserModel]:
         except:
             return None
 
-    def get_user_by_email(self, email: str) -> Optional[UserModel]:
+    def get_user_by_email(
+        self, email: str, oauth_user: bool = False
+    ) -> Optional[UserModel]:
+        try:
+            conditions = (
+                (User.email == email, User.oauth_sub.is_null())
+                if not oauth_user
+                else (User.email == email)
+            )
+            user = User.get(conditions)
```

Comment:
This condition is wrong. You are passing the tuple directly to the query which is causing a `peewee.OperationalError`. I suppose it should work prepending the `*` :
```python
user = User.get(conditions)
```